core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -61,6 +61,9 @@ private class ClientEndpoint(
 
    private val lostMasters = new HashSet[RpcAddress]
    private var activeMasterEndpoint: RpcEndpointRef = null
+   private val waitAppCompletion = conf.get("spark.submit.waitAppCompletion", "false") == "true"
```

Comment:
We can use conf.getBoolean here?

Reply:
Thanks @prakharjain09 , updated to use conf.getBoolean.